cve_bin_tool/data_sources/osv_source.py

```diff
@@ -256,7 +298,15 @@ def format_data(self, all_cve_entries):
 
     async def get_cve_data(self):
         await self.update_ecosystems()
-        await self.fetch_cves()
+
+        # skip OSV if connection fails
+        try:
+            await self.fetch_cves()
+        except Exception as e:
+            LOGGER.debug(f"Error while fetching OSV CVEs : {e}")
+            LOGGER.warning("Unable to fetch OSV CVEs, skipping OSV.")
```

Comment:
I think this should probably be an error rather than a warning, but I'll probably open a separate issue for that rather than waiting since this is currently causing issues in CI.